app/workers/workflow_retired_count_worker.rb

```diff
@@ -3,23 +3,20 @@ class WorkflowRetiredCountWorker
 
   sidekiq_options queue: :data_high
 
-  attr_reader :workflow
+  sidekiq_options congestion: {
+    interval: 10,
```

Comment:
It seems this worker can be fired multiple times from the `workflows_controller#retire_subjects` route (think aggregation engine). I assume once every 10 seconds with a cancel if exceeding will be ok...thoughts?